### ISSUE TITLE
Any privacy policy ?

### ISSUE THREAD

**Issue by FaizKhan5** [AllanWang/Frost-for-Facebook#1293]
what type of data you people can access ?

why there is no playstore version ?

**Comment by AllanWang** [comment]
I used to have a play store version, but it was removed since this app violates the terms and conditions. An app is not allowed to redirect traffic to a single website, which all third party Facebook apps do. It's true that other alternatives are on the play store, but there's nothing I can personally do about that.

In terms of privacy, the one I had while it was on the play store is [here](https://www.allanwang.ca/frost-for-facebook/privacy-policy.html). I will move it into the github repo and link it in the future.

In short, I do not collect any identifiable information. You may optionally allow bug reports to be sent to me (as you can see in the issues in this repo), and that's it. I also don't use any unnecessary permissions, and as this app is open, you are free to investigate and compile it yourself if you feel it is necessary.
